### DISCUSSION TITLE
Replace ILMerge with dnMerge

### DISCUSSION THREAD

**Discussion by rasta-mouse** [Mayyhem/SharpSCCM#42] [Ideas]
The assembly produced by ILMerge is 2,444KB.  Merging the assembly with dnMerge instead reduces this to 1,098KB thanks to the LZMA compression.  This would make it a little easier to utilise over C2, particularly over lower bandwidth channels.

**Comment by Mayyhem** [comment]
Totally agree @rasta-mouse. I have some time to work on this coming up and will make sure this gets updated.

**Comment by Mayyhem** [comment]
@rasta-mouse I just merged this code into main -- thanks a ton for pointing me to dnMerge!
